packages/common/src/index.ts

```diff
@@ -69,22 +74,27 @@ export const fetchPassport = (
  *
  * @param address string
  * @param communityId string
+ * @param round Round
  * @returns
  */
 export const submitPassport = (
   address: string,
-  communityId: string
+  communityId: string,
+  round: Round
 ): Promise<Response> => {
```

Comment:
since we already did the switch to decide which communityId to use, let's receive here the api key as well